src/cascadia/TerminalApp/PowershellCoreProfileGenerator.cpp

```diff
@@ -10,81 +10,320 @@
 #include "Utils.h"
 #include "DefaultProfileUtils.h"
 
-using namespace ::TerminalApp;
+#include <winrt/Windows.ApplicationModel.h>
+#include <winrt/Windows.Management.Deployment.h>
 
-// Legacy GUIDs:
-//   - PowerShell Core       574e775e-4f2a-5b96-ac1e-a2962a402336
+#include <appmodel.h>
+#include <shlobj.h>

```

Comment:
I don't exactly want their existence to pollute the rest of T-App. What do you think?

Reply:
Oh that's a good point - maybe leave a note about that since I'm usually keen to just grab any `include <foo>` lines I see and move them to pch without thinking

Reply:
i shall. ty.